_posts/2023-12-07-semantic-options-benchmarks.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title:  "Semantic Search with OpenSearch: Architecture options and Benchmarks"
+authors:
+ - sean-zheng-amazon
+ - zane-neo
```

Comment:
Not seeing an author short_name defined in _authors for this author either.

Reply:
added zan's bio and pic